envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/reliability/Toxiproxy.kt

```diff
@@ -1,13 +1,13 @@
 package pl.allegro.tech.servicemesh.envoycontrol.reliability
 
-import com.pszymczyk.consul.infrastructure.Ports
 import eu.rekawek.toxiproxy.Proxy
 import org.testcontainers.junit.jupiter.Testcontainers
 import pl.allegro.tech.servicemesh.envoycontrol.config.BaseEnvoyTest.Companion.consul
 import pl.allegro.tech.servicemesh.envoycontrol.config.BaseEnvoyTest.Companion.network
 import pl.allegro.tech.servicemesh.envoycontrol.config.containers.ToxiproxyContainer
 import pl.allegro.tech.servicemesh.envoycontrol.config.containers.ToxiproxyContainer.Companion.internalToxiproxyPort
 import pl.allegro.tech.servicemesh.envoycontrol.config.testcontainers.GenericContainer.Companion.allInterfaces
+import pl.allegro.tech.servicemesh.envoycontrol.utils.Ports
```

Comment:
Don't you need to free ports if they are taken from `Ports` class? Same thing in
```
envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoycontrol/EnvoyControlTestApp.kt
envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/consul/ConsulSetup.kt
```

Reply:
After our talk I decided to remove freeing ports altogether - it was a bit problematic to decide when ConsulSetup should use static ports.